### PR TITLE
[API-5339]  Update changes.rst and bump master version to dev one

### PR DESCRIPTION
**Pull request by jettify**
# BATCH SCORING PULL REQUEST
* Updated Changes.rst file
* Bumped master version to unreleased one.

### CODE REVIEW

**Comment by jettify** [comment]
@datarobot/batch-scoring please reveiw

File: datarobot_batch_scoring/__init__.py [hunk 1]
```diff
@@ -1 +1 @@
-__version__ = '1.13.3'
+__version__ = '1.13.4a1'
```
Comment (Axik) [inline datarobot_batch_scoring/__init__.py]: I'd say it's `1.14.0a1`. This functionality is definitely worth bumping _minor_ version.

Comment (jettify) [reply datarobot_batch_scoring/__init__.py]: fixed

**Comment by jettify** [comment]
Updated readme with compatibility information.

**Pull request merged**